Dockerfile

```diff
@@ -0,0 +1,54 @@
+ENV LANG en_US.UTF-8  
+
+# Install EPEL repo, yum necessary packages for the build without docs, clean all caches (no DNF)
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && yum -y install --setopt=tsflags=nodocs nodejs tar sudo git memcached postgresql-devel postgresql-server libxml2-devel libxslt-devel patch gcc-c++ openssl-devel gnupg curl which net-tools libyaml-devel autoconf readline-devel libffi-devel bzip2 automake libtool bison sqlite-devel ; yum clean all
+
+# Systemd cleanups, setup volumes
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+rm -f /lib/systemd/system/multi-user.target.wants/*;\
+rm -f /etc/systemd/system/*.wants/*;\
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*;\
+rm -f /lib/systemd/system/anaconda.target.wants/*;
+VOLUME [ "/sys/fs/cgroup" ]
+
+# POSTGRESQL INIT
+# Do not call a login "-" su, resets the ENV
+RUN su postgres -c 'initdb -D /var/lib/pgsql/data'
+
+## 2. RVM
+# Download and install RVM, setup environment, install ruby/gems, clean all
+# Note: RVM uses yum to bring missing pre-reqs
+
+RUN /usr/bin/curl -sSL https://rvm.io/mpapis.asc | gpg2 --import - && /usr/bin/curl -sSL https://get.rvm.io | rvm_tar_command=tar bash -s stable && source /etc/profile.d/rvm.sh && echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc && /bin/bash -l -c "rvm requirements ; rvm install ruby 2.2.3 ; rvm use 2.2.3 --default ; gem install bundler rake ; gem install nokogiri -- --use-system-libraries ; rvm cleanup all ; yum clean all ; rvm disk-usage all"
+
+# GIT clone and prepare services (shallow clone)
+RUN mkdir /manageiq && git clone --depth 1 https://github.com/ManageIQ/manageiq /manageiq
+
+# Change workdir to clone, prepare database, start it, run setup, shutdown and cleanup all
+WORKDIR /manageiq
+#COPY docker_setup bin/docker_setup
+RUN su postgres -c "pg_ctl -D /var/lib/pgsql/data start" && sleep 5 && su postgres -c "psql -c \"CREATE ROLE root SUPERUSER LOGIN PASSWORD 'smartvm'\"" && su postgres -c "pg_ctl -D /var/lib/pgsql/data stop"
+RUN su postgres -c "pg_ctl -D /var/lib/pgsql/data start" && /bin/bash -l -c "/usr/bin/memcached -u memcached -p 11211 -m 64 -c 1024 -l 127.0.0.1 &" && /bin/bash -l -c "bin/setup" && su postgres -c "pg_ctl -D /var/lib/pgsql/data stop" && pkill memcached && /bin/bash -l -c "rvm cleanup all ; rm -vf $(/usr/local/rvm/bin/rvm gemdir)/cache/* ; rvm disk-usage all"
+
+# Copy evmserver startup script and systemd evmserverd unit file
+COPY evmserver.sh bin/evmserver.sh
+COPY evmserverd.service /usr/lib/systemd/system/evmserverd.service
+
+# Enable services on systemd
+RUN systemctl enable evmserverd memcached postgresql
+
+# Expose required container ports
+EXPOSE 3000 4000
```

Comment:
@bdunne  I don't understand why ssh should be opened (we do not install ssh in the docker immage) , getting a console into container is done differently.
Postgres eventually will reside in a different container, but I do not see anything wrong in opening it's port  even in this monolithic.
http & https will be handled together with the httpd config - I don't see a need for httpd in the developer use case , but I'm sure it will be handled for the upstream & downstream official images.